Rules/CommonScripts/ChatCommands/UtilityCommands.as

```diff
@@ -0,0 +1,66 @@
+#include "AbstractCommand.as"
```

Comment:
``AbstractCommand.as`` doesn't exist. Remove this line to get rid of a warn in the console